backend/src/main/resources/application.properties

```diff
@@ -10,6 +10,9 @@ quarkus.http.test-ssl-port=0
 # only enable the next line in order to allow CORS request from any domain for all modes (dev and prod, etc), e.g. for UI frontend development (see details in README.md), when you can't use the dev mode
 # quarkus.http.cors.origins=*
 
+# Jolokia endpoint inside Kafka Connect images
+jolokia.client.uri=http://dbzui-connect:8778/jolokia
```

Comment:
I  don't think this is correct. Shouldn't the hostname be derived from the connect cluster (based on the `{cluster}` URL parameter)? The only option should be the port name. Maybe we should extend the Kafka Connect cluster config for this instead?

Reply:
@rk3rn3r Applied the changes, but the testcontainers test seems to fail. It works fine on the UI.

Reply:
Hey @ani-sha! Great to see progress here! Let me have a look

Reply:
@ani-sha I have looked at the CI tests and commented below what might be causing the issue.